recipe/meta.yaml

```
@@ -249,7 +264,7 @@ outputs:
         - wrapt >=1.11.0
         # TF-API needs to move in sync
         - tensorboard >=2.17,<2.18
-        - keras >=3.0
+        - keras >=3.2
```

```suggestion
        - keras >=3.2.0
```
https://github.com/tensorflow/tensorflow/blob/v2.17.0/tensorflow/tools/pip_package/setup.py#L118